build/admission/deployment.yaml

```diff
@@ -30,7 +30,6 @@ spec:
             - --ca-cert-file=/admission.local.config/certificates/ca.crt
             - --webhook-namespace=kubeedge
             - --webhook-service-name=kubeedge-admission-service
-            - --alsologtostderr
```

Comment:
If we use klog, there should be `alsologtostderr` in options, maybe there're have some errors when initiating klog? @tangming1996 @Catherine-monk

Reply:
![image](https://github.com/kubeedge/kubeedge/assets/109125240/dd1c2c2e-40a3-41d2-b930-aa7076f6d3b0)

Yes, I see that the documentation originally supported klog, but it may have been removed later, so here we delete the alsologtostderr parameter from the startup command, otherwise the service will fail to start.